deepspeed/pt/fp16_optimizer.py

```diff
@@ -314,6 +351,10 @@ def state_dict(self):
         state_dict['clip_grad'] = self.clip_grad
         return state_dict
 
+    def refresh_fp32_params(self):
```

Comment:
where is this function called?

Reply:
This will be called from the client function in the case where non-DeepSpeed checkpoint is loaded to initialize the model.